src/pages/Login/Login.js

```diff
@@ -37,48 +58,32 @@ function Login() {
           이메일계정 하나로 편리하게 이용해보세요!
         </p>
         <div className="imagesWrap">
-          <img
-            src="https://accounts.kakao.com/assets/weblogin/techin/retina/banner_login1-81752909ec9c815794b13ccff4d17d7d3e81f431585ecec30ee7a8055785c74f.png"
-            alt="노트북이미지"
-          />
+          <img src="/images/Login/Branch_login.png" alt="노트북이미지" />
         </div>
       </section>
       <section className="rightWrap">
         <article>
           <h1 className="loginTitle">Email</h1>
           <form className="loginWrap">
             <input
-              className={userId ? 'loginIdInnerText' : 'loginId'}
+              className={userId ? 'loginInput ' : 'loginInputChange'}
```

Comment:
scss 파일 리뷰 참고해보시고, 다음처럼 작성해볼 수 있습니다.

```suggestion
              className={`loginInput${userId ? '' : ' lightBorder'}`}
```

Reply:
<img width="1058" alt="스크린샷 2021-12-03 오후 4 22 27" src="https://user-images.githubusercontent.com/82882076/144561845-9332ad0c-9d89-4837-a243-ab9027d6b492.png"

<img width="328" alt="스크린샷 2021-12-03 오후 4 22 50" src="https://user-images.githubusercontent.com/82882076/144561890-f1c1ba3e-600f-4ca8-a81e-c36f17e3d91b.png">
>

sass 수정사항은 수정했어요!

input의 클래스도 수정해봤는대  값이 없을 때 는 투명으로 되어있다가 입력하면 선이 보이게 되었어요.
생각했던 레이아웃이 아니여서 더 생각해 보겠습니다.

Reply:
래영님 코드대로 수정해보니까 적용 잘 됐어요! 감사합니다.
```
 className={`loginInput${userId ? '' : ' lightBorder'}`}
  className={`loginInput${userId ? '' : 'lightBorder'}`}
```
제가 했던건 공백을 안주고 했더니 적용이 안됐었나봐요!